oldabe/money_in.py

```diff
@@ -168,26 +171,33 @@ def calculate_incoming_investment(payment, price):
 
 
 def calculate_incoming_attribution(
-    email, incoming_investment, posterior_valuation
+        email, incoming_investment, posterior_valuation, attributions
 ):
     """
     If there is an incoming investment, find out what proportion it
     represents of the overall (posterior) valuation of the project.
     """
+    nondilutable_shares = [a.share for a in attributions.values() if not a.dilutable]
+    nondilutable_proportion = sum(nondilutable_shares)
+    dilutable_proportion = 1 - nondilutable_proportion
```

Comment:
Instead of getting the nondilutable total and subtracting from 1, maybe we should just get the dilutable total.